src/System.Data.SqlClient/src/ApiCompatBaseline.net45.txt

```diff
@@ -0,0 +1,4 @@
+MembersMustExist : Member 'System.Data.SqlClient.SqlConnectionStringBuilder.ConnectRetryCount.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
Looks like these were added in 4.5.1.  @saurabh500 do you want another intermediate version of the contract or push these up to the 4.1 version?